core/citrus-api/src/main/java/com/consol/citrus/util/IsJsonPredicate.java

```diff
@@ -0,0 +1,22 @@
+package com.consol.citrus.util;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests if a string represents a Json. An empty string is considered to be a
+ * valid Json.
+ */
+public class IsJsonPredicate implements Predicate<String> {
+
+    public static IsJsonPredicate INSTANCE = new IsJsonPredicate();
```

Comment:
add a private constructor, this will prevent anyone from unintentionally creating a new instance. I personally prefer accessors too (aka `static getInstance()`), but that's probably a matter of taste.

Reply:
private constructor is certainly required to fullfill singleton pattern. I also added a getInstance for your taste.

Reply:
![clap](https://c.tenor.com/ZeDln5QiCHkAAAAM/clap-leonardo-de-caprio.gif)